core/src/labels/labelMesh.h

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+#include "gl/typedMesh.h"
+#include "labels/label.h"
+
+#include <memory>
+#include <vector>
+
+namespace Tangram {
+
+class LabelMesh : public TypedMesh<Label::Vertex> {
+public:
+    LabelMesh(std::shared_ptr<VertexLayout> _vertexLayout, GLenum _drawMode);
+
+    virtual ~LabelMesh();
+
+    void addLabel(std::unique_ptr<Label> _label);
+
+    std::vector<std::unique_ptr<Label>>& getLabels() {
+        return m_labels;
+    }
+
+protected:
+    std::vector<std::unique_ptr<Label>> m_labels;
```

Comment:
Hmm, yes, storing objects in-place would be good. I think one could use a templated version. 
The question is how to get the items as Label instances. 
Either a combination of getLabelsSize() + getLabel(i) or a forEachLabel(LabelVisitor) comes to my mind.

I'll think about it